test/JDBC/upgrade/latest/schedule

```diff
@@ -268,6 +268,7 @@ BABEL-SP_SPROC_COLUMNS_100-dep
 BABEL-3000-dep
 Test-sp_helprole-dep
 Test-sp_helprolemember-dep
+TestDatetime-sql
```

Comment:
In addition to the latest schedule file, please add appropriate tests to at least 15.2 schedule file. You may need to split some tests (one for the latest and another for the previous versions).